package.json

```diff
@@ -14,6 +14,7 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@chakra-ui/icons": "^1.0.14",
```

Comment:
We want to avoid adding dependencies whenever possible, and since we are already using react icons this dependency should be removed.  You can remove it by simply deleting this line in package.json.